### PR TITLE
Latency metrics for rnn-t

### PR DESCRIPTION
**Pull request by yuekaizhang**
Usage
```
python3 tools/latency_metrics.py \
  --config ${config} \
  --ckpt ${ckpt} \
  --tag ${tag} \
  --wavscp ${wavscp} \
  --alignment ./ali_word_small.txt \
  --chunk_size ${chunk_size} \
  --font ${font_path} \
  --dict ${dict_path} \
  --result_dir ${result_dir} \
  --gpu -1 \
  --model_type transducer
```
E.g. 

image

### CODE REVIEW

File: tools/latency_metrics.py [hunk 1]
```diff
@@ -231,9 +294,8 @@ def f(name_idx=name_idx):
             axes[-1].plot(time, samples)
 
             # i.e., RESULT_DIR/BAC009S0768W0342_LTD_P90_120ms.pdf
```
Comment (xingchensong) [inline tools/latency_metrics.py:296]: "RESULT_DIR/BAC009S0768W0342_LTD_P90_120ms.pdf"  -> "RESULT_DIR/LTD_P90_120ms_BAC009S0768W0342.pdf"

**Review by xingchensong — APPROVED** [review]

**Comment by lling-2021** [comment]
/ali_word_small.txt怎么得到的呀

**Comment by xingchensong** [comment]
> /ali_word_small.txt怎么得到的呀

https://huggingface.co/yuekai/wenet-librispeech-trimtail/tree/main

**Comment by yuekaizhang** [comment]
> > /ali_word_small.txt怎么得到的呀
> 
> https://huggingface.co/yuekai/wenet-librispeech-trimtail/tree/main

This is for English. Also, it's alignment generated from baseline model rather than GT.

**Comment by lubacien** [comment]
hello, could you provide details on how the alignment file should look like exactly? with kaldi, I am able to create an alignment file with the show-alignment command with the following structure:

utt_id_1 [ number ........................................................................................................ number ] [ number ............. number] 
utt_id_1 corresponding_phoneme                                                                                                corresponding_phoneme2 ...
..

This does not seem to be the expected format, as it is not working

**Pull request merged**